wormhole-connect/src/views/v2/Redeem/index.tsx

```diff
@@ -461,7 +500,12 @@ const Redeem = () => {
 
   // Main CTA button which has separate states for automatic and manual claims
   const actionButton = useMemo(() => {
-    if (!isTxComplete && !isTxRefunded && !isTxFailed && !isAutomaticRoute) {
+    if (
+      !isTxComplete &&
+      !isTxRefunded &&
+      !isTxFailed &&
+      (isTxDestQueued || !isAutomaticRoute)
```

Comment:
maybe this would be easier to read if we checked for what we _do_ expect it to be, rather than what we do not expect it to be?

isn't it basically just `(state === TransferState.Attested && !isAutomatic) || isTxDestQueued`?

I realize this was already like this before your change... just thinking out loud here.

Reply:
i agree that this is ugly, but going to leave it for now since we know that it works. we should probably refactor the redeem view in the future as it's becoming quite complex